### PR TITLE
Happy Sunday and Happy Father's day to all father out there

### PR DESCRIPTION
**Pull request by Anipal1**
# WELCOME TO DEVOPS-EASY-LEARNING PULL REQUEST SYSTEM 
# Description of the change

> get is the deal

## hard work will always pay

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## addition to initial code

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

### CODE REVIEW

File: anipal [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This git world is getting intresting by the day
+Hard work can really pay 
+Let keep pushing it, is a matter of time 
+We will all get there?
```
Comment (lucydaniel) [inline anipal:4]: we will all get where?

Comment (Anipal1) [reply anipal:4]: I mean getting a job as DevOps Engineer

**Comment by lucydaniel** [comment]
ok

**Review by lucydaniel — APPROVED** [review]
good Job Mr P

**Pull request merged**